frame/staking/src/tests.rs

```diff
@@ -2990,6 +2990,60 @@ mod offchain_phragmen {
 			})
 	}
 
+	#[test]
+	fn signed_result_can_be_submitted_regardless_active_era() {
```

Comment:
Maybe this test is not very useful, it was in order to test that solution could be submitted for current_era.

Reply:
I think it is not necessary. 

What matters more is a test to ensure if the era that comes with the solution != current_era, then we reject. 

But I think we already have that.